src/site/content/en/blog/local-fonts/index.md

```diff
@@ -14,15 +14,14 @@ description: >
   The Local Fonts API enumerates the user's installed local fonts and provides low-level access to
```

Comment:
Nit: Why is it called "Local Font Access API" and sometimes "Local Fonts API"?

Reply:
Fixed with https://github.com/GoogleChrome/web.dev/pull/8261/commits/8403b51739efa1c428d22e3d3ad5cea0b423b287.